.aws/.ebextensions/01_nginx_custom.config

```diff
@@ -0,0 +1,60 @@
+files:
```

Comment:
probably easy for nginxers but I think the config could use some comments

Reply:
Even better: all this .ebextension folder will be gone. 
We are not using it automatically, and SRE is doing this job, better reduce code conflicts